src/renderer/components/layout/page-layout.scss

```diff
@@ -54,13 +59,19 @@
       width: var(--width);
       min-width: var(--min-width);
       max-width: var(--max-width);
-      padding: calc(var(--spacing) * 2);
+      padding: var(--wrapper-padding);
       margin: 0 auto;
 
       .content-navigation {
         > ul {
           position: fixed;
           width: var(--nav-width);
+          height: calc(100vh - var(--header-height) - var(--wrapper-padding) - var(--bottom-bar-height));
```

Comment:
Why do we need so much fixed height calculations?
Couldn't it just work with some proper `display: grid` + `overflow: auto`?